src/test/xattr_bench.cc

```diff
@@ -152,15 +152,6 @@ int main(int argc, char **argv) {
   argv_to_vec(argc, (const char **)argv, args);
 
   global_init(0, args, CEPH_ENTITY_TYPE_CLIENT, CODE_ENVIRONMENT_UTILITY, 0);
-  common_init_finish(g_ceph_context);
```

Comment:
as a convention, we call `common_init_finish()` after `global_init()` to finish the initialization. as the accompanied comment puts

>  * This includes things like starting threads, initializing libraries that
>  * can't handle forking, and so forth.

and if we take it as a black box, we should not assume what it is doing under the hood. the `g_ceph_context`could be used somewhere else by ObjectStore in some unknown way. so we'd better leave as it is even we know ObjectStore does not use anything initialized by `common_init_finish()`.